05/app.js

```diff
@@ -1 +1,4 @@
-const numbers = [1, 2, 3, 4, 5, 6, 7];
\ No newline at end of file
+const numbers = [1, 2, 3, 4, 5, 6, 7];
+
+const sumOfEvenNumbers = numbers.filter(number => number%2 === 0).reduce((prev, curr) => prev + curr);
```

Comment:
Uważałbym na tworzenie takich łańcuchów. 
Wygląda to efektownie, ale zmniejsza czytelność kodu i zwiększa możliwość popełnienia błędu.
Dodatkowo w `.reduce()` brakuje drugiego parametru, który rozwiązuje problem błędu przy pustej tablicy.